bindings/ocaml/hacl-star.opam

```diff
@@ -22,5 +22,5 @@ run-test: [
   ["dune" "runtest" "-p" name "-j" jobs]
 ]
 available: [
-  os-family != "bsd"
+  os-family != "bsd" & arch != "ppc64" & arch != "ppc32" & arch != "x86_32"
```

Comment:
@msprotz This was [added at some point](https://github.com/ocaml/opam-repository/pull/18009) in opam-repository because the PowerPC build was failing. Do you think it's still necessary with the other changes in this PR? In particular it was failing because -mavx -mavx2 were being passed, which should no longer be the case.

Reply:
this should definitely be removed and it should work now